src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,432 @@
+                      <v-text-field
+                        ref="signingPersonFamilyNameRef"
+                        filled
+                        class="item"
+                        label="Last Name"
+                        id="person__last-name"
+                        v-model="signingPerson.familyName"
+                        :rules="lastNameRules"
+                        />
+                    </div>
+                  </v-form>
+                </v-col>
+              </v-row>
+
+              <v-row no-gutters>
+                <v-col cols="12" sm="3" class="pr-4">
+                  <label class="resolution-signature-vcard-title">Date Signed</label>
+                </v-col>
+                <v-col cols="12" sm="9" class="pt-4 pt-sm-0">
+                  <DatePickerShared
+                    ref="signatureDatePickerRef"
+                    title="Date Signed"
+                    :nudgeRight="40"
+                    :nudgeTop="85"
+                    :minDate="signatureDateMinStr"
+                    :maxDate="signatureDateMaxStr"
+                    :inputRules="signatureDateRules"
+                    @emitDateSync="onSigningDateSync"
+                  />
+                </v-col>
+              </v-row>
+            </v-card>
+
+        </section>
+      </v-card>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, HelpSectionIF, ResourceIF,
+  FormIF, SigningPersonIF, EmptySigningPersonIF, CreateResolutionIF } from '@/interfaces/'
+import { DateMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class SpecialResolutionForm extends Mixins(DateMixin) {
+  @Getter getResource!: ResourceIF
+  @Getter getBusinessFoundingDate!: string
+  @Getter getCurrentDate!: string
+  @Getter getCurrentJsDate!: string
+
+  @Action setResolution!: ActionBindingIF
+  @Getter getcreateResolution!: CreateResolutionIF
+  @Getter getComponentValidate!: boolean
+
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePickerShared,
+    signatureDatePickerRef: DatePickerShared,
+    signingPersonFormRef: FormIF,
+  }
+
+  // Date properties
+  protected resolutionDateText = ''
+  protected signatureDateText = ''
+
+  readonly MAX_RESOLUTION_TEXT_LENGTH = 1000
+  protected resolutionText = ''
+
+  protected signingPerson: SigningPersonIF = null
+
+  /** Validation rule for individual name fields */
+  readonly firstNameRules = this.nameRules('First Name')
+  readonly middleNameRules = this.nameRules('Middle Name', false)
+  readonly lastNameRules = this.nameRules('Last Name')
+
+  get helpSection (): HelpSectionIF {
+    return this.getResource.changeData?.specialSpecialResolution?.helpSection
```

Comment:
This is used by HelpSection component. If that component does not already handle falsy (undefined) then you should have a fallback here (like line 242 maybe?).